### ISSUE TITLE
Assigning tasks to contact

### ISSUE THREAD

**Issue by cherrielam** [PeaceGeeksSociety/salesforce#1]

**Comment by cherrielam** [comment]
Major development postponed until we decide to move forward with Volunteers For Salesforce

**Comment by PeaceGeeksProjects** [comment]
@cherrielam Can you be more specific?  Tasks and Jobs are 2 different objects in SF.

Currently, Volunteers are assigned Jobs against which the can track time.

Tasks are assigned to SF Users.  What are the benefits of using Tasks rather than continue with Jobs?

**Comment by cherrielam** [comment]
@PeaceGeeksProjects I'm envisioning tasks as more specific details/instructions of what needs to be done, literally a task list from a team lead to a volunteer. Right now Jobs don't really do that. The idea of Tasks assigned to Volunteer contact was to have the possibility of displaying such Tasks on the Volunteer Personal Site, where a Volunteer could log in and see all tasks relevant to them at once. Whereas Jobs is for logging hours specific to a type of task, right?

eg. Content Volunteer may be assigned a Task of "Writing a blog about extremist media in North America"
Content Volunteer logs into Personal Site and sees "Writing a blog about extremist media in North America" listed under "Assigned tasks," Volunteer Personal Site becomes a place where Volunteer can see assigned tasks in one place.

Once task is complete, Volunteer can track hours worked under the appropriate Job (type of task), eg. 6 hours under MR-BP-Content Creation.

I understand that this process doesn't exist yet or hasn't been developed yet, so this might be developed in other ways. Or alternatively, it may be deemed that it's unwieldy to assign tasks to Volunteer contacts in SF. I just suggested this as a way to incentivize usage of the Volunteer Personal Site; otherwise I don't imagine enforcement of its usage being a well-adopted process considering that not many volunteers care about tracking hours and this would rely on a lot of capacity for enforcement which is not something we currently have in place (but I recognize we could adopt if we deem this a useful function)

Some questions for consideration:
- Is it possible to have "Tasks" associated with "Jobs" such that Volunteers are assigned "Jobs" but the Volunteer Manager/Team Lead can write a note (describing the Task) as to what that Job entails?
- Would it be possible for the Volunteer to mark the Job as complete (in their Personal Site) and that would automatically add those hours to their time tracking?

**Comment by PeaceGeeksProjects** [comment]
If you could not get the TLs to enter Jobs, you think you will get them to enter SUB-jobs???  I honestly have my doubts.  In the Job description filed you can have a list of tasks for sure.  I think we should start there and evolve if & when the need arises
